components/AdCard.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <v-alert border="bottom">
+    I'm a card of type ad with a link {{ card.name }}.
```

Comment:
ad cards don't have a name, maybe a url can be shown as an example?

Reply:
Done 👍 Let's make them look nice after this PR is merged

Reply:
Yes. Focus has been on UI for the product cards, but today I've solved some major things there so I guess we can start updating the extra cards components next.